compendium/DeclarativeServices/src/manager/ConfigurationNotifier.cpp

```diff
@@ -183,22 +186,32 @@ namespace cppmicroservices
             auto registry = mgr->GetRegistry();
             auto logger = mgr->GetLogger();
             auto configNotifier = mgr->GetConfigNotifier();
-            auto managers = mgr->GetManagers();
-
             try
             {
                 auto compManager = std::make_shared<ComponentManagerImpl>(newMetadata,
                                                                           registry,
                                                                           bundle.GetBundleContext(),
                                                                           logger,
                                                                           asyncWorkService,
-                                                                          configNotifier,
-                                                                          managers);
+                                                                          configNotifier);
                 if (registry->AddComponentManager(compManager))
                 {
-                    managers->push_back(compManager);
-                    compManager->Initialize();
-                }
+                    auto extension = extensionRegistry->Find(bundle.GetBundleId());
+                    if (extension)
```

Comment:
use [initializers for if statements](https://en.cppreference.com/w/cpp/language/if)